build/README.md

```diff
@@ -0,0 +1,33 @@
+# Building the Ride Framework
+
+Ride is based on the core Java/Maven concepts of modularity and extensibility.  Because of this, you may build all of Ride from the parent pom located in the /build directory, or and of the sub modules from their root location, where their specific pom resides.
+
+## Syntax and Location
+
+The build command follow the standard maven syntax.  However, because of the inclusion of GPG signing in the dependencies, in order to build locally and unsigned version of Ride, you need to add an additional flag ```-Dgpg.skip```
```

Comment:
* follow -> follows
* to build locally and unsigned version of Ride -> to build an unsigned version of Ride locally (note especially 'and' to 'an')

Reply:
Fixed.